src/components/MemberSearchPage/FilterTable.js

```diff
@@ -0,0 +1,64 @@
+import { Collapse, Divider, Input, Typography } from 'antd';
+import { withTheme } from 'emotion-theming';
```

Comment:
We should avoid using 'emotion-theming' since Julien is working on the build and has to remove that lib. I'd suggest to hard code the values that you need for now.